src/config/settings/rest_framework.py

```diff
@@ -1,3 +1,50 @@
 """
 Django Rest Framework configuration.
 """
+from datetime import timedelta
+
+REST_FRAMEWORK = {
+
+'DEFAULT_AUTHENTICATION_CLASSES': [
+    'rest_framework_simplejwt.authentication.JWTAuthentication',
+]
+}
+SIMPLE_JWT = {
+    "ACCESS_TOKEN_LIFETIME": timedelta(minutes=5),
+    "REFRESH_TOKEN_LIFETIME": timedelta(days=1),
+    "ROTATE_REFRESH_TOKENS": False,
```

Comment:
Maybe we should set "ROTATE_REFRESH_TOKENS" to `True`. 
If `True`, then the API users could refresh their refresh token also.
https://django-rest-framework-simplejwt.readthedocs.io/en/latest/settings.html#rotate-refresh-tokens

Reply:
Resolved in a irl conversation